src/core/utils/index.js

```diff
@@ -174,3 +174,9 @@ export const redirectToUrl = event => {
     document.body.removeChild(anchor)
   } catch (e) {}
 }
+
+export const takeSubmitEventData = (event) => {
+  const { landingId } = event.detail || {}
+
+  return landingId && { response_id: landingId }
```

Comment:
Why not make the `response_id` camel case (`responseId`)?

Reply:
I was thinking to match [API](https://developer.typeform.com/responses/reference/retrieve-responses/) schema

Reply:
so that if you make api request, you get `response.response_id` which is equal to `event.response_id` - less cognitive overhead

Reply:
Makes sense 👍